src/main/kotlin/snyk/code/annotator/SnykAnnotator.kt

```diff
@@ -47,25 +51,27 @@ abstract class SnykAnnotator(private val product: ProductType): ExternalAnnotato
                     return@forEach
                 }
                 if (!textRange.isEmpty) {
-                    val annotationMessage = "${issue.annotationMessage()} (Snyk)"
+                    val annotationMessage = issue.annotationMessage()
```

Comment:
What's the reason to remove `(Snyk)`?

Reply:
I moved the code that appends it to `src/main/kotlin/snyk/common/lsp/Types.kt`, it makes it a bit more obvious what we render for the different product lines